genienlp/tasks/almond_task.py

```diff
@@ -1,10 +1,3 @@
-#
-# Copyright (c) 2019, The Board of Trustees of the Leland Stanford Junior University
-# All rights reserved.
-#
-# Redistribution and use in source and binary forms, with or without
-# modification, are permitted provided that the following conditions are met:
-#
```

Comment:
???
Please don't alter copyright notices, that's literally illegal.

Reply:
I didn't change it! It was just copied partially. Fixed now...